workbench_access.services.yml

```diff
@@ -1,4 +1,10 @@
 services:
   plugin.manager.workbench_access.scheme:
     class: Drupal\workbench_access\WorkbenchAccessManager
-    arguments: ['@container.namespaces', '@cache.discovery', '@module_handler', '@entity_type.manager', '@state']
+    arguments: ['@container.namespaces', '@cache.discovery', '@module_handler', '@entity_type.manager', '@workbench_access.user_section_storage', '@workbench_access.role_section_storage', '@config.factory', '@current_user']
+  workbench_access.user_section_storage:
```

Comment:
@agentrickard this PR does split them into two separate services - here are the service definitions.

Thanks for the merge.